### PR TITLE
Upload resume

### PR DESCRIPTION
**Pull request by alemannosara**
We can still not see it but we can upload one and it goes to the database.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ before_script:
   - ./cc-test-reporter before-build
 script: 
   - ./gradlew build
-  - travis_wait 30 ./gradlew connectedCheck jacocoTestReport
+  - travis_wait 30 ./gradlew connectedCheck jacocoTestReport -i
```
Comment (fmutlu) [inline .travis.yml]: What's the purpose of the parameter -i here ?

File: app/src/main/java/ch/epfl/sweng/thegramginers/Helpers/UploadHelper.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            requestPermissions(activity, new String[]{android.Manifest.permission.READ_EXTERNAL_STORAGE, android.Manifest.permission.WRITE_EXTERNAL_STORAGE}, selectRequest);
+        } else {
+            Intent intent = new Intent();
+            intent.setType(type);
+            intent.setAction(Intent.ACTION_GET_CONTENT);
+            startActivityForResult(activity, Intent.createChooser(intent, "Select your " + toSelect), selectRequest, null);
+        }
+    }
+
+    public static Bitmap getCroppedAndRoundedBitmap(Bitmap source) {
```
Comment (fmutlu) [inline app/src/main/java/ch/epfl/sweng/thegramginers/Helpers/UploadHelper.java]: Maybe this function should be moved to ImageHelper for more consistency.

File: .travis.yml (hunk 1 above)
Comment (alemannosara) [reply .travis.yml]: It shows what tests succeeds/fails with more details on the Travis CI build.

File: app/src/main/java/ch/epfl/sweng/thegramginers/Helpers/UploadHelper.java (hunk 2 above)
Comment (alemannosara) [reply app/src/main/java/ch/epfl/sweng/thegramginers/Helpers/UploadHelper.java]: Yes, you're right!

**Comment by alemannosara** [comment]
@fmutlu Thanks for your review. I will give a try and if I see that it is too complicated to deal with those UIDevices, I will get rid of them.

**Comment by alemannosara** [comment]
@fmutlu as you're the scrum master for this week, do you agree that the coverage will be lower and I merge? I am deleting useless files on another PR so it will get a bit higher.

**Pull request merged**